config/nvim/keys.vim

```diff
@@ -0,0 +1,7 @@
+" Remap Leader Key
+" ---------------------------------------------------------------------------
+let mapleader = "\<Space>"
+
+" Deoplete Tab Completion
+" ---------------------------------------------------------------------------
+inoremap <expr><TAB>  pumvisible() ? "\<C-n>" : "\<TAB>"
```

Comment:
Unfortunately, Tab completion won't work because `<TAB>` is already mapped to something else. Looks like the UltiSnips plugin maps `<TAB>`as the expand trigger here: https://github.com/docwhat/dotfiles/blob/master/config/nvim/plugins.vim#L422